models/reviews.models.js

```diff
@@ -169,3 +170,11 @@ exports.removeComment = (comment_id) => {
       }
     });
 };
+
+exports.selectAPIs = () => {
+  return readFile("./endpoints.json", "utf-8")
+    .then((result) => {
+      return JSON.parse(result);
+    })
+    .catch((err) => console.log(err));
+};
```

Comment:
Nice use of readFile :D We do not need the model for this, though, this can live in the controller & the model can be omitted